src/backend/api/Fusion.Resources.Api/FusionEvents/OrgProjectHandler.cs

```diff
@@ -0,0 +1,81 @@
+﻿using Fusion.Events;
+using Fusion.Integration.Org;
+using Fusion.Resources.Database;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Logging;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Fusion.Resources.Api
+{
+    public class OrgProjectHandler : ISubscriptionHandler
+    {
+        private static readonly JsonSerializerOptions serializerOptions;
+
+        private readonly ResourcesDbContext db;
+        private readonly IProjectOrgResolver orgResolver;
+        private readonly ILogger<OrgProjectHandler> logger;
+
+        static OrgProjectHandler()
+        {
+            serializerOptions = new JsonSerializerOptions
+            {
+                PropertyNameCaseInsensitive = true,
+                IncludeFields = true
+            };
+            serializerOptions.Converters.Add(new JsonStringEnumConverter());
+        }
+
+        public OrgProjectHandler(ResourcesDbContext db, IProjectOrgResolver orgResolver, ILogger<OrgProjectHandler> logger)
+        {
+            this.db = db;
+            this.orgResolver = orgResolver;
+            this.logger = logger;
+        }
+        
+        public async Task ProcessMessageAsync(MessageContext ctx, string body, CancellationToken cancellationToken)
+        {
+            var payload = JsonSerializer.Deserialize<Events.Org.OrgSubscriptionEvent>(body, serializerOptions);
```

Comment:
I think you can simplify this to:
var payload = ctx.GetBody<OrgSubscriptionEvent>();